lib/rules/lint-no-childless-elements.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/*
+ Disallows childless elements
+
+ ```
+{{! good }}
```

Comment:
I would remove this entire comment. The documentation file already provides the good and bad examples. It's not worth duplicating the list again here.